File: search/evaluation-services/src/main/java/ir/ac/iust/dml/kg/evaluation/services/web/controller/WelcomeController.java
package ir.ac.iust.dml.kg.evaluation.services.web.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/")
public class WelcomeController {
  @RequestMapping({"/", ""})
  public String index() {
    return "Greetings from Evaluation Services! " +
        "<br/>" +
        "Select one of these options:" +
        "<br/>" +
        "<a href='/swagger-ui.html'>All public services</a>";
  }
}

File: update/update_manager/src/main/java/ir/ac/iust/dml/kg/search/feedback/web/commons/controller/WelcomeController.kt
/*
 * Farsi Knowledge Graph Project
 * Iran University of Science and Technology (Year 2017)
 * Developed by Majid Asgari.
 */

package ir.ac.iust.dml.kg.search.feedback.web.commons.controller

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class WelcomeController {
  @RequestMapping(path = arrayOf("/", ""))
  fun index() = """
    Greetings from Spring Boot!
    <br/>
    Select one of these options:
    <br/>
    <a href='/swagger-ui.html'>All public services</a>
    """
}